lib/ruby_lsp/ruby_lsp_rails/runner_client.rb

```diff
@@ -96,6 +96,14 @@ def initialize
         raise InitializationError, @stderr.read
       end
 
+      sig { params(server_addon_path: String).void }
+      def register_server_addon(server_addon_path)
```

Comment:
Why do we need both `inherited` and `register_server_addon`?

Reply:
🤔 perhaps we don't.... @vinistock ?

Reply:
> We need to do some testing first on a partial rollout.

We can manage that on the Tapioca side though, right?

Reply:
Currently, there's no way to perform a partial rollout. But we could inform addons if experimental features are enabled so that they can decide whether to do something or not.

That would give you some control.

Reply:
As I said above, with this implementation, rollout can be controlled by the individual addons through the trigger of `register_server_addon`. It's not an issue.